README.md

```diff
@@ -7,6 +7,7 @@ The `master` branch has the "incomplete" solution; please read the [instructions
 ## Prerequisites
 
 * [dotnet SDK 3.1.1 or higher](https://dotnet.microsoft.com/download) The .NET Core SDK including CLI tools
+* [FAKE 5](https://fake.build/fake-gettingstarted.html#Install-FAKE) F# build tool installed as a global tool
```

Comment:
This is incorrect. You do not need to install FAKE 5 as a pre-req, it's done as part of dotnet local tools now.